src/routes/medias.rs

```diff
@@ -72,7 +72,12 @@ pub fn upload(
                     .map(|ext| format!(".{}", ext))
             })
             .unwrap_or_default();
-        let dest = format!("static/media/{}{}", GUID::rand().to_string(), ext);
+        let dest = format!(
+            "{}/{}{}",
+            CONFIG.media_directory,
+            GUID::rand().to_string(),
```

Comment:
is this supposed to be deterministic?

Reply:
Is it deterministic? It seems to come up with new GUIDs each piece we upload.

Reply:
aah, never mind, i was thinking of something else
(the deterministially random id we use per release for *our* assets)